### ISSUE TITLE
[Feature Request] Default to streamer for quotes setting

### ISSUE THREAD

**Issue by itsjesski** [crowbartools/Firebot#2336]
**Describe the solution you'd like**
Some people are confused as to why they need to put a name when adding a quote, because on their channel only the streamer is ever quoted. This seems to be a fairly common thing. 

We should add a setting somewhere that makes it so that all quotes default to the streamer name, and in turn that would make it so that no name would need to be provided when entering a new quote.

**Comment by olliesouthie** [comment]
Seconding this! This is not common syntax among other bots for quotes so mods frequently get it wrong and it's especially frustrating that its usage can't be modified because it's a system command. Even just the ability to edit system command inputs in the same detail as custom commands (with a "restore to default" option I guess, to protect anyone from irreparably screwing it up) would be great. Thanks! :)

**Comment by NafenX** [comment]
I agree with this wholeheartedly! I'm also confused as to why there is a "Quote Creator" and a "Quote Originator" box in the Add Quote effect, when the Quotes section (and clicking on individual quotes to edit/delete) only shows info of the "Quote Author". I feel like there's too many different words being used there 😅
